virt/virtual_machines/virt-managing-vms-openshift-pipelines.adoc

```diff
@@ -0,0 +1,51 @@
+:_content-type: ASSEMBLY
+[id="virt-managing-vms-openshift-pipelines"]
+= Managing virtual machines with OpenShift Pipelines
+include::_attributes/common-attributes.adoc[]
+:context: virt-managing-vms-openshift-pipelines
+
+toc::[]
+
+xref:../../cicd/pipelines/understanding-openshift-pipelines.adoc#understanding-openshift-pipelines[Red Hat OpenShift Pipelines] is a Kubernetes-native CI/CD framework that allows developers to design and run each step of the CI/CD pipeline in its own container.
```

Comment:
You can use attributes for the OpenShift Pipelines references throughout this PR. Pasting the relevant section from the attributes file:

//pipelines
:pipelines-title: Red Hat OpenShift Pipelines
:pipelines-shortname: Pipelines
:pipelines-ver: pipelines-1.8
:tekton-chains: Tekton Chains
:tekton-hub: Tekton Hub
:pac: Pipelines as Code